devel/fortio/cmd/grpcping/pingsrv.go

```diff
@@ -0,0 +1,163 @@
+// Copyright 2017 Istio Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+// Adapted from istio/proxy/test/backend/echo with error handling and
+// concurrency fixes and making it as low overhead as possible
+// (no std output by default)
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"net"
+	"os"
+	"time"
+
+	"istio.io/istio/devel/fortio"
+
+	context "golang.org/x/net/context"
+
```

Comment:
nit: unnecessary separation of import blocks.

other nit: here and elsewhere, you are using import blocks of the order: stdlib, istio, external. elsewhere in istio, it is typically: stdlib, external, istio.